src/cloud/client.rs

```diff
@@ -5,12 +5,13 @@ use std::io;
 use std::path::PathBuf;
 use std::time::Duration;
 
+use anyhow::Context;
 use surf::http::auth::{AuthenticationScheme, Authorization};
 
 use crate::options::CloudOptions;
 use crate::platform::config_dir;
 
-const EDGEDB_CLOUD_BASE_URL: &str = "https://free-tier0.ovh-us-west-2.edgedb.cloud";
+const EDGEDB_CLOUD_DEFAULT_DNS_ZONE: &str = "aws.edgedb.cloud";
```

Comment:
We still want the ability to override the complete API host address with `EDGEDB_CLOUD_BASE_URL`.